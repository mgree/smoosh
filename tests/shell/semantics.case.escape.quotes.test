foo=\"
echo -n Literal: ''
case "$foo" in
    \" ) echo OK ;;
    * ) echo NOT OK ;;
esac
echo -n Unquoted: ''
case "$foo" in
    $foo ) echo OK ;;
    * ) echo NOT OK ;;
esac
echo -n Quoted: ''
case "$foo" in
    "$foo" ) echo OK ;;
    * ) echo NOT OK ;;
esac
