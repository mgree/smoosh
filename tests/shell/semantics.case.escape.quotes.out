Literal: OK
Unquoted: OK
Quoted: OK
